documentation/source/tutorials/pages/registration_to_template.rst

```diff
@@ -0,0 +1,293 @@
+.. _registration-to-template:
+
+Tutorial 2: Registering T2 images to the PAM50 template
+#######################################################
+
+This tutorial demonstrates a multi-step pipeline to register an anatomical MRI scan to the PAM50 Template. While T2 images are used for this tutorial, each step is applicable to multiple contrast types (T1, T2, T2*).
+
+Before starting this tutorial
+*****************************
+
+1. Read through the following pages to familiarize yourself with key SCT concepts:
+
+    * :ref:`pam50`: An overview of the PAM50 template's features, as well as context for why the template is used.
+    * :ref:`warping-fields`: Background information on the image transformation format used by the registration process.
+    * :ref:`qc`: Primer for SCT's Quality Control interface. After each step of this tutorial, you will be able to open a QC report that lets you easily evaluate the results of each command.
+
+2. Download and unzip `sct_course_london20.zip <https://osf.io/bze7v/?action=download>`_.
+3. Open a terminal and navigate to the ``sct_course_london20/single_subject/data/t2/`` directory.
+
+----------
+
+.. _segmentation-section:
+
+Step 1: Segmenting the spinal cord
+**********************************
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/registration-pipeline-1.png
+   :align: center
+   :figwidth: 100%
+
+First, we will run the ``sct_deepseg_sc`` command to segment the spinal cord from the anatomical image.
+
+.. code:: sh
+
+   sct_deepseg_sc -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
+
+   # Input arguments:
+   #   - i: Input image
+   #   - c: Contrast of the input image
+   #   - qc: Directory for Quality Control reporting. QC reports allow us to evaluate the segmentation slice-by-slice
+
+   # Output files/folders:
+   #   - t2_seg.nii.gz: 3D binary mask of the segmented spinal cord
```

Comment:
TODO: Fix the commented annotations (input/output arguments) because they prevent the codeblock from being used for copy/paste.